docs/spl-docs/evaluation-functions/informational-functions.md

```diff
@@ -87,7 +86,7 @@ This function returns `TRUE` if the given value is an integer.
 
 ### Usage
 
-`isint` can be used with `eval` and `where` commands to check if a field's value is an integer. This is particularly useful in conditional expressions and for filtering records based on integer field values.
+`isint` can be used with `eval` and `where` commands to check if a field's value is an integer. This is useful for conditional checks and for filtering records that contain an integer value in any field.
```

Comment:
This is a little misleading, because it sounds like using `isint(<field>)` would be useful for filtering records that have an integer in any field; but in fact it would be filtering records that have an integer in the specified field. You can just copy the structure of your description for `isnum`:
> This comes in handy for conditional checks and for filtering records that have any numeric value in a field.

Then you just have to slightly change it for `isint` and `isbool`